premium/usage.go

```diff
@@ -307,6 +342,9 @@ func (u *BatchUpdater) TeamName() string {
 }
 
 func (u *BatchUpdater) HasQuota(ctx context.Context) (bool, error) {
+	if u.awsMarketPlaceClient != nil {
```

Comment:
In AWS marketplace there's no concept of reaching a quota or a limit? You just get billed for more 💵 ?

Reply:
Yes, there is no sort of concept of a limit or quota, it is a problem with AWS where you can't limit the spend of an account, which can lead to run away bills